modules/react/breadcrumbs/lib/BreadcrumbsMenu.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from 'react';
+import {createContainer, ExtractProps} from '@workday/canvas-kit-react/common';
+import {Menu, useMenuModel} from '@workday/canvas-kit-react/menu';
+
+export interface BreadcrumbsProps {
+  /**
+   * The contents of the Breadcrumbs. Can be `Breadcrumbs` children or any valid elements.
+   */
+  children: React.ReactNode;
+}
+
+export const BreadcrumbsMenu = createContainer()({
+  displayName: 'Breadcrumbs.Menu',
+  modelHook: useMenuModel,
+  subComponents: {
+    Card: (elemProps: any) => (
```

Comment:
Should this be `CardProps`?

Reply:
Unfortunately, no one of this prop is not exported, so I can use it. `ExtractProps` doesn't work as well.

Reply:
ahhh, ok!

Reply:
Q: I see both of these working locally. What am I missing?

```ts
Card: (elemProps: ExtractProps<typeof Card, never>) => (
```

```ts
Card: (elemProps: ExtractProps<typeof Menu.Card, never>) => (
```

If those work, I would go with `Menu.Card` to have one less import.